tests/install_upgrade_operators/crypto_policy/conftest.py

```
@@ -119,3 +135,124 @@ def services_to_check_connectivity(hco_namespace, admin_client):
+        namespace=hco_namespace.name,
+        client=admin_client,
+        pod_selector={"matchLabels": {"app.kubernetes.io/component": "kubevirt-console-plugin"}},
+        ingress=[{"ports": [{"protocol": "TCP", "port": CONSOLE_PLUGIN_SERVICE_PORT}]}],
+        policy_types=["Ingress"],
+    ):
+        yield
+
+
+@pytest.fixture(scope="module")
+def pqc_status_by_service(
+    enabled_aaq, workers_utility_pods, worker_node1, cnv_services_with_template, console_plugin_test_network_policy
```

Ruff (0.15.12)

[warning] Unused function argument: `enabled_aaq`

(ARG001)